packages/elements/src/slider/elements/slider.ts

```diff
@@ -0,0 +1,1481 @@
+import {
+  CSSResultGroup,
+  ControlElement,
+  PropertyValues,
+  StyleMap,
+  TemplateResult,
+  WarningNotice,
+  css,
+  html,
+  nothing
+} from '@refinitiv-ui/core';
+import { customElement } from '@refinitiv-ui/core/decorators/custom-element.js';
+import { property } from '@refinitiv-ui/core/decorators/property.js';
+import { query } from '@refinitiv-ui/core/decorators/query.js';
+import { state } from '@refinitiv-ui/core/decorators/state.js';
+import { Ref, createRef, ref } from '@refinitiv-ui/core/directives/ref.js';
+import { styleMap } from '@refinitiv-ui/core/directives/style-map.js';
+
+import '@refinitiv-ui/phrasebook/locale/en/slider.js';
+import { TranslateDirective, translate } from '@refinitiv-ui/translate';
+
+import type { NumberField } from '../../number-field/index.js';
+import '../../number-field/index.js';
+import { VERSION } from '../../version.js';
+import { Direction, NumberFieldName, SliderDataName, SliderPreviousDataName } from '../constants.js';
+import { clamp, countDecimalPlace, isDecimalNumber, preventDefault } from '../utils.js';
+import './slider-marker.js';
```

Comment:
let's remove this line as it's covered by `import { SliderMarker } from './slider-marker.js';` already.